L1Trigger/L1THGCal/python/hgcalTriggerNtuples_cfi.py

```diff
@@ -40,7 +47,7 @@
     eeSimHits = cms.InputTag('g4SimHits:HGCHitsEE'),
     fhSimHits = cms.InputTag('g4SimHits:HGCHitsHEfront'),
     bhSimHits = cms.InputTag('g4SimHits:HcalHits'),
-    isSimhitComp = cms.bool(False)
+    isSimhitComp = cms.bool(True)
```

Comment:
Better to keep it false by default because some samples we use don't have simhit information